charts/kyverno-policies/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
```

Comment:
@eddycharly do we need to test the helm upgrade to see upgrade is successful ?

Reply:
Should be non-issue for Helm v3 : https://helm.sh/docs/topics/charts/#the-apiversion-field.

Reply:
@prateekpandey14 du you mean upgrading a release that was deployed with helm v2 ?

Reply:
Helm v2 been EOL a while now I think

Reply:
Yup, there's an helm plugin to convert from v2 to v3 if necessary.